GLMakie/test/glmakie_refimages.jl

```diff
@@ -65,6 +65,7 @@ end
         markersize=size,
         axis = (; scenekw = (;limits=Rect3f(Point3(0), Point3(1))))
     )
+    update_cam!(ax.scene, Point3f(2.224431, 2.224431, 2.128731), Point3f(0.5957, 0.5957, 0.50000006))
```

Comment:
I hard coded the camera setup here (and below) to avoid test errors. I don't know if we want to keep this or update refimgs so I'm pointing it out. The difference is tiny.